index.html

```diff
@@ -7,5 +7,17 @@
 </head>
 <body>
 
+<header>Header A</header>
+
+<article> <p>B: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci ex, fugiat necessitatibus nostrum numquam praesentium quis sapiente. Aliquam dolorum ea eum expedita facere iure neque vero voluptate. Atque commodi cupiditate debitis dolorum eius fugiat, ipsum iste maxime natus qui quisquam tempore. Autem deserunt, doloribus ea, mollitia natus nemo obcaecati officiis optio sunt, suscipit tempore temporibus? A accusantium atque consequatur cum delectus eaque eos facilis fugiat illum minima neque omnis, quia ratione repellat sed! Ad inventore magnam perferendis repellat voluptas! Ad amet assumenda autem cum ea et exercitationem fuga ipsum magni natus obcaecati qui ratione reiciendis saepe ut, veniam, vero. Ab aliquid atque dolore dolorem est exercitationem facere hic in incidunt, minima molestias nihil quia, quibusdam quis recusandae rem saepe sequi similique vel veniam. Ad eum incidunt laudantium magni minima, nihil nostrum odit, officiis optio quam, quas quod sed voluptate? Aliquam animi corporis cum exercitationem fugit inventore rem, reprehenderit saepe voluptate! </p></article>
```

Comment:
Tip: Je kunt in Webstorm en Intelij in de bovenste balk via View > Active editor > Soft Wrap de code aanpassen dat het leesbaar is in 1 winow verdeeld over meerder regels ipv 1 lange  regel.